ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -3,38 +3,20 @@
 {-# LANGUAGE FlexibleContexts         #-}
 {-# LANGUAGE NamedFieldPuns           #-}
 {-# LANGUAGE OverloadedStrings        #-}
+{-# LANGUAGE RecordWildCards          #-}
```

Comment:
We tend to avoid this one in the ledger, but maybe you guys are fine with it?

Reply:
greping for `RecordWildCards` on ouroboros gives 169 results :)
but i promise we will not move it to ledger :)